translations/en-us.yml

```diff
@@ -1349,6 +1352,7 @@ registries:
         submit: 'Submit'
         success: 'Successfully submitted moderator decision.'
         failure: 'Failed to submit moderator decision.'
+        noActionAvailable: 'The withdraw request is pending approval by registration admins. No moderator action is available at this moment.'
```

Comment:
if we add the `noActionAvailable` into the each/else block, this will need some adjusting. I think `No moderator action is available at this moment.` should suffice, since the moderator should be able to see that it is pending withdraw/embargo termination on the `<RegistriesState>`